lib/siteAdapter.coffee

```diff
@@ -158,7 +158,11 @@ siteAdapter.origin = {
       type: 'GET'
       dataType: 'json'
       url: "/#{route}"
-      success: (page) -> done null, page
+      success: (page, code, xhr) ->
+        if route is 'system/sitemap.json'
+          done null, { data: page, lastModified: Date.parse(xhr.getResponseHeader('Last-Modified') )}
```

Comment:
We are guessing if a lazy origin implementation didn't attach the header then elsewhere you detect NaN and call that zero.

Reply:
This is handled in [neighborhood.coffee line 51](https://github.com/fedwiki/wiki-client/blob/42e9af8ecca71989f2539ba10521a6b4498f939b/lib/neighborhood.coffee#L51), part of this PR.